src/status_im/contexts/wallet/common/screen_base/create_or_edit_account/view.cljs

```diff
@@ -1,35 +1,46 @@
 (ns status-im.contexts.wallet.common.screen-base.create-or-edit-account.view
-  (:require [quo.core :as quo]
+  (:require [clojure.string :as string]
+            [quo.core :as quo]
             [quo.theme]
             [react-native.core :as rn]
             [react-native.safe-area :as safe-area]
             [status-im.common.floating-button-page.view :as floating-button-page]
             [status-im.constants :as constants]
             [status-im.contexts.wallet.common.screen-base.create-or-edit-account.style :as style]
             [utils.i18n :as i18n]
-            [utils.re-frame :as rf]))
+            [utils.re-frame :as rf]
+            [utils.string]))
 
 (defn view
   [{:keys [page-nav-right-side placeholder account-name account-color account-emoji
            on-change-name
            on-change-color
            on-change-emoji section-label
            bottom-action-label bottom-action-props
-           custom-bottom-action watch-only?]} & children]
+           custom-bottom-action watch-only? top-left-icon]} & children]
   (let [{window-width :width} (rn/get-window)
         footer                (if custom-bottom-action
                                 custom-bottom-action
                                 [quo/button
                                  (merge {:size 40
                                          :type :primary}
                                         bottom-action-props)
-                                 (i18n/label bottom-action-label)])]
+                                 (i18n/label bottom-action-label)])
+        emoji?                (and account-emoji (fn? on-change-emoji))
+        account-name-initials (when-not emoji?
+                                (->> (string/split (if (string/blank? account-name)
+                                                     placeholder
+                                                     account-name)
+                                                   #" ")
+                                     (map (fn [segment]
+                                            (utils.string/get-initials segment 1)))
+                                     (apply str)))]
```

Comment:
Maybe we have a helper that extracts initials from multiple whitespace-separated words?